docs/docs/notification-center/react-components.md

```diff
@@ -230,6 +231,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>vi</code> (Vietnamese)</li>
           <li><code>zu</code> (Zulu)</li>
           <li><code>zh</code> (Chinese)</li>
+
```

Comment:
Not sure if it is because of this, but CI looks like it is complaining for this empty line before the `ul` element.

Reply:
Yep, seems like a doc linter in action